### PR TITLE
feat/cleaned up formatting, added clarity, removed unnecessary parts

### PR DESCRIPTION
**Pull request by cmccormack**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/cplusplus/index.md [hunk 1]
```diff
@@ -1,102 +1,81 @@
...
-* It is a version of C that includes Object-Oriented elements, including classes and functions.
-
-* It is considered one of the biggest programming languages, as you can see in the following image:
-![Img](http://static1.businessinsider.com/image/59deb30392406c21008b6148-1200/for-bonus-points-heres-the-chart-showing-these-languages-relative-popularity.jpg)
+- C++ is a general purpose programming [object-oriented programming](https://en.wikipedia.org/wiki/Object-oriented_programming) (OOP) language which has been used since the 1990's
+- It was designed as a thesis paper by [Bjarne Stroustrup](https://en.wikipedia.org/wiki/Bjarne_Stroustrup) under the name "C with classes", which later became C++.
+- It is a version of C that includes OOP elements, including classes and functions.
+- It is considered one of the biggest programming languages, as you can see in the following image:
+![programming language popularity](http://static1.businessinsider.com/image/59deb30392406c21008b6148-1200/for-bonus-points-heres-the-chart-showing-these-languages-relative-popularity.jpg)
```
Comment (thecodingaviator) [inline guide/english/cplusplus/index.md]: @cmccormack I think we can put this in unnecessary parts too, we do not absolutely need this image to be here and this article could do just fine without it

Comment (cmccormack) [reply guide/english/cplusplus/index.md]: Are you referring to the last bullet only or the entire section?

Comment (thecodingaviator) [reply guide/english/cplusplus/index.md]: Just the graphic

Comment (cmccormack) [reply guide/english/cplusplus/index.md]: Removed, thanks

**Review by thecodingaviator — APPROVED** [review]
The formatting LGTM

**Review by moT01 — APPROVED** [review]
👍 LGTM @cmccormack 👍

**Pull request merged**